data/releases/4.14.1.md

```diff
@@ -0,0 +1,172 @@
+---
+kind: compiler
+version: 4.14.1
+date: 2022-12-20
+intro: |
+  This page describes OCaml version **4.14.1**, released on
+  Dec 20, 2022. Go [here](/releases) for a list of all releases.
+
+  This is a bug-fix release of [OCaml 4.14.0](/releases/4.14.0).
+highlights: |
+  - Bug fixes for 4.14.1
+---
+
+## Installation Instructions
+
+The base compiler can be installed as an opam switch with the following commands:
+```bash
+opam update
+opam switch create 4.14.1
+```
+
+### Configuration options
+
+The configuration of the installed opam switch can be tuned with the
+following options:
+
+- ocaml-option-afl: set OCaml to be compiled with afl-fuzz instrumentation
```

Comment:
```suggestion
- ocaml-option-afl: sets OCaml to be compiled with `afl-fuzz` instrumentation
```

Should all these items be in monospace?
Like `ocaml-option-afl`: ?
Also, since it’s a singular thing, it will take the singular for of the verb. In this case “sets” instead of “set.”

Reply:
Since the option are technical terms rather than English words, I think it makes sense for them to be in monospace.